src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -328,6 +373,227 @@ public void SetFocus(bool focused)
+        }
+
+        /// <summary>
+        /// Get and Sets the scroll position of the page.
+        /// </summary>
+        /// <returns>The class Point object with X, Y coordinates.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public Point ScrollPos
+        {
+            get
+            {
+                Point p;
+                Interop.ChromiumEwk.ewk_view_scroll_pos_get(_realHandle, out p.X, out p.Y);
+                return p;
+            }
+            set
+            {
+                Interop.ChromiumEwk.ewk_view_scroll_set(_realHandle, value.X, value.Y);
+            }
+        }
+
+        /// <summary>
+        /// Scrolls the webpage of view by x and y.
+        /// </summary>
+        /// <param name="p">The class Point object with X, Y coordinates.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public void ScrollBy (Point p)
+        {
+            Interop.ChromiumEwk.ewk_view_scroll_by(_realHandle, p.X, p.Y);
+        }
+
+        /// <summary>
+        /// Searches and highlights the given text string in the document.
+        /// </summary>
+        /// <param name="text">The text to find.</param>
+        /// <param name="options">The options to find.</param>
+        /// <param name="maxMatchCount">The maximum match count to find, unlimited if 0.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool FindText (string text, FindOptions options, int maxMatchCount)
+        {
+            return Interop.ChromiumEwk.ewk_view_text_find(_realHandle, text, (Interop.ChromiumEwk.FindOptions)options, maxMatchCount);
+        }
+
+        /// <summary>
+        /// Enumeration for Http Method.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public enum Method
+        {
+            /// <summary>
+            /// Get.
+            /// </summary>
+            Get,
+            /// <summary>
+            /// Head.
+            /// </summary>
+            Head,
+            /// <summary>
+            /// Post.
+            /// </summary>
+            Post,
+            /// <summary>
+            /// Put.
+            /// </summary>
+            Put,
+            /// <summary>
+            /// Delete.
+            /// </summary>
+            Delete, 
+        }
+
+        /// <summary>
+        /// Requests loading of the given request data.
+        /// </summary>
+        /// <param name="url">The uniform resource identifier to load.</param>
+        /// <param name="method">The http method.</param>
+        /// <param name="headers">The http headers.</param>
+        /// <param name="body">The http body data.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool SetUrlRequest (string url, Method method, string headers, string body)
```

Comment:
`hearders` should be Dictionary<string, string> instead of string.
And we should convert Dictionary to Eina_Hash* in order to invoke C native API.

Reply:
Done